lib/vc.js

```diff
@@ -82,24 +88,43 @@ async function issue(options = {}) {
   });
 
   // run common credential checks
-  await _checkCredential({credential: options.credential});
+  const {credential} = options;
+  if(!credential) {
+    throw new Error('"credential" property is required for issuing.')
+  }
+  _checkCredential(credential);
 
-  return jsigs.sign(options.credential, {purpose, documentLoader, ...options});
+  return jsigs.sign(credential, {purpose, documentLoader, ...options});
 };
 
 /**
- * Verify a credential.
+ * Verifies a credential.
+ *
+ * @param {object} [options={}]
+ * @param {object} [options.credential] - Verifiable credential
+ * @param {CredentialIssuancePurpose} [options.purpose]
+ * @param {function} [options.documentLoader]
+ *
+ * @returns {Promise<{verified: boolean, results: *[], error: Error}>}
  */
 async function verify(options = {}) {
   const {credential} = options;
+
+  if(_isPresentation(credential)) {
```

Comment:
This is strange ... a VP isn't a VC nor vice versa.

Reply:
> This is strange ... a VP isn't a VC nor vice versa.

Sure, I know. And there's a separate `verifyPresentation()` function. It's just that I know that devs will try to use `verify()` with a Presentation, so that's just there to catch it.

I don't feel super strongly about it, so if you don't like it, happy to remove it.

Reply:
I think passing a VP as `credential` should be an error. If we think people will want to call `verify` with a VP, then we should accept a `presentation` parameter as well -- and throw an error if both `presentation` and `credential` are passed. Then we can also remove the `verifyPresentation` call and just use `verify`.

Reply:
I like the idea of being able to pass either `credential` or `presentation` to `verify()`. I'll change it to that!